.github/workflows/build-and-test-other.yaml

```diff
@@ -22,17 +22,22 @@ jobs:
 
         include:
         - arch: "armel"
-          distro: "wheezy"
-          base_url: "http://archive.debian.org/debian"
-          cflags: "-std=gnu99 -O2 -mthumb -mthumb-interwork -march=armv4t"
+          distro: "stretch"
+          base_url: "http://deb.debian.org/debian/"
+          cflags: "-O2 -mthumb -mthumb-interwork -march=armv4t"
+          additional_packages: |
+            http://ftp.us.debian.org/debian/pool/main/c/cmake/cmake-data_3.13.2-1~bpo9+1_all.deb \
+            http://ftp.us.debian.org/debian/pool/main/r/rhash/librhash0_1.3.3-1+b2_armel.deb \
+            http://ftp.us.debian.org/debian/pool/main/libu/libuv1/libuv1_1.34.2-1~bpo9+1_armel.deb \
+            http://ftp.us.debian.org/debian/pool/main/c/cmake/cmake_3.13.2-1~bpo9+1_armel.deb
```

Comment:
Just for my knowledge about cmake 3.13:
- did we update cmake version to this version for any specific feature?
- given the features we are using with the next PRs, in theory, do you know which is the minimum cmake version we need?

Reply:
Indeed. We need 3.13 at least for `target_link_options` which is used or coverage with Clang. As this is the latest version for stretch-backports, I aligned everything with it. If we need newer features, we will need to use a cross-compiler for armel (which I think we should do anyway).